src/components/search/search-result-section.module.scss

```diff
@@ -3,8 +3,6 @@
 
 .section {
   &:not(:last-of-type) {
-    padding-bottom: 1em;
-    margin-bottom: .5em;
-    border-bottom: 1px solid $color-plum-400;
+    margin-bottom: 2em;
```

Comment:
This one is not about your code, it's OK, but as most of the site uses `rem` units I guess we should be careful about consistency.

Reply:
I was reviewing some of @beep's code and it seems that `rem` is used on font sizes, and `em` for spacing. I am planning to do a deeper dive into that, not sure the reasoning.

Reply:
That's weird indeed, most code I found in files I've worked on were using `rem`... This definitely deserves some standardization!

Reply:
We don’t really have a standard here! My general approach has been treating the spacing on an element as relative to the element itself, whereas font sizing’s relative to the root. But the distinction’s pretty academic/esoteric; either works, really.

Reply:
I find the `em` approach, while sometimes more challenging, allows fine-tuning of elements behavior/styling, particularly on responsive websites. Avoided me many media queries and overloaded CSS sheets on relatively complex websites like e-shops, so I personally like it better.
Next week I will probably work on the new homepage, we could try to take what's already here and make a sensible - but not too strict - standard for the project. For sure this would help newcomers being more confident about their first features, while reducing the final CSS size which would be great!

Reply:
> I find the em approach, while sometimes more challenging, allows fine-tuning of elements behavior/styling, particularly on responsive websites.

Hm! I’m not aware of any _layout_ benefits to `rem` vs `em`—they’re [both relative units](https://blog.typekit.com/2011/11/09/type-study-sizing-the-legible-letter/)—or how it’d reduce CSS file size. Would love to hear more about that, if you’d care to share more! (Happy to have that conversation in Slack, btw!)

But! All that aside! I really do like the idea of using the homepage work to establish some decent standards, great idea 🙌